docs/setup_gdb.md

```diff
@@ -8,520 +8,394 @@ GDB Tutorial
-```
-┌──stats_tests.cpp─────────────────────────────────────────────────────────────┐
-│   42            vector<double> data;                                         │
-│   43            data.push_back(1);                                           │
-│   44            data.push_back(2);                                           │
-│   45            data.push_back(3);                                           │
-│   46                                                                         │
-│  >47            assert(sum(data) == 6);                                      │
-│   48                                                                         │
-│   49            cout << "PASS!\n";                                           │
-│   50          }                                                              │
-│   51                                                                         │
-│   52          // Add the test function implementations here.                 │
-│                                                                              │
-│                                                                              │
-└──────────────────────────────────────────────────────────────────────────────┘
-native process 1209 In: test_sum_small_data_set        L47   PC: 0x555555557a3d
-(gdb) n
+### Step over
+Enter `n` AKA "Next" AKA "Step Over" a few times until you reach the highlighted line of code.
+
+```
 (gdb) n
+   ┌──main.cpp─────────────────────────────────────────────────────────────────┐
+   │10        return total;                                                    │
+   │11      }                                                                  │
+   │12                                                                         │
+   │13      int main() {                                                       │
+B+ │14        vector<double> data;                                             │
+   │15        data.push_back(10);                                              │
+   │16        data.push_back(20);                                              │
+   │17        data.push_back(30);                                              │
+  >│18        cout << "sum(data) = " << sum(data) << endl;                     │
+   │19      }                                                                  │
+   │20                                                                         │
+   │21                                                                         │
+   │22                                                                         │
+   └───────────────────────────────────────────────────────────────────────────┘
+native process 95465 In: main                                L18   PC: 0x400cd2 
+
 (gdb) n
 (gdb) n
 (gdb) n
 (gdb) n
-(gdb) refresh
-(gdb)
 ```
+{: data-highlight="11,19-22" }
+
+<div class="primer-spec-callout info" markdown="1">
+**Pro-tip:** Hit Return to repeat your previous command.
```

Comment:
```suggestion
**Pro-tip:** Hit <kbd>Enter</kbd> to repeat your previous command.
```
Guessing Enter is more familiar than Return for Windows users.
The \<kbd> styling is just a thought.